notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -5043,6 +5043,38 @@ pride goes before a fall [Anakin was boasting of his powers when he tried to jum
 mobile life support device [Anakin's badly mutilated body was treated and covered in a black armored suit to keep him alive.]
 
 
+movie: Primer (2004)
+====================
+
+:: Title
+Primer
+
+:: Description
+Primer is a 2004 American independent psychological science fiction film about
+the accidental discovery of time travel. The film was written, directed,
+produced, edited and scored by Shane Carruth, who also stars with David
+Sullivan.
+
+:: Date
+2004-10-08
+
+:: References
+https://en.wikipedia.org/wiki/Primer_(film)
+
+:: Ratings
+4 <paul>
+
+:: Choice Themes
+time travel [The story is all about two engineers accidentally discovering a peculiar form of time travel and there consequences thereof.]
+
+:: Major Themes
+friendship [Abe and Aaron's divergent views on what to do now that they could time travel put a strain on their friendship.]
+
+:: Minor Themes
+husband and wife [Abe interacted with his wife, Rachel. Later he saved her from getting shot.]
+what if I met my double [Abe from the future took measures to keep his past self from traveling into the future.]
```

Comment:
https://www.themeontology.org/theme.php?name=what%20if%20I%20met%20myself%20from%20a%20different%20time
?

Reply:
Ah, that's a good fit. I replaced **what if I met my double* by it.